src/tracer/MOM_tracer_advect.F90

```diff
@@ -123,7 +123,8 @@ subroutine advect_tracer(h_end, uhtr, vhtr, OBC, dt, G, GV, US, CS, Reg, x_first
   x_first = (MOD(G%first_direction,2) == 0)
 
   ! increase stencil size for Colella & Woodward PPM
-  if (CS%usePPM .and. .not. CS%useHuynh) stencil = 3
+! if (CS%usePPM .and. .not. CS%useHuynh) stencil = 3
```

Comment:
Simply increasing the stencil size may fix the problems with the OBCs, probably by avoiding working inward from wide halos to avoid extra halo updates, but it will also add pass var calls to all cases, and will have a potentially significant performance impact on tracer-rich global cases (like ESMs).  I think we should be cautious about this change and explore whether some added logic could allow us to retain the narrower stencil in some cases.

Reply:
The additional tests that Theresa has done have convinced me that we are fixing the problem for the right reason, and not masking a different problem that we need to look into, so we should go ahead and accept this PR.

Reply:
Are we keeping this comment for any reason?  Or can it be removed?

Reply:
I would prefer that we keep this as a comment.  I am not completely certain that this particular change is the right thing to do in all circumstances, and keeping this comment could be a helpful reminder when we come back to reexamine this question later.